include/openmc/particle.h

```diff
@@ -181,6 +181,15 @@ class Particle {
     int delayed_group; //!< particle delayed group
   };
 
+  //! Saved ("banked") state of primary source particle, for particle restart
+  struct RestartBank {
+    Position r;
+    Direction u;
+    double E;
+	double wgt;
+  };
```

Comment:
TAB character messing up indentation. Replace with spaces.

```suggestion
  struct RestartBank {
    Position r;
    Direction u;
    double E;
    double wgt;
  };
```